db/migrate/20220202002459_create_rule_groups_add_columns_to_rules.rb

```diff
@@ -0,0 +1,26 @@
+class CreateRuleGroupsAddColumnsToRules < ActiveRecord::Migration[6.1]
+  PHONY_BENCHMARK = Xccdf::Benchmark.find_or_create_by!(
+    ref_id: 'phony_ref_id',
+    version: 'v0.0.0',
+    title: 'phony title',
+    description: 'phony description'
+  )
```

Comment:
I'm pretty sure this is unnecessary, the migration won't create any rows that would break the integrity of the DB.